zboss/README.rst

```diff
@@ -6,3 +6,9 @@ ZBOSS Zigbee stack
 The |NCS|'s Zigbee stack uses ZBOSS – a portable, high-performance Zigbee software protocol stack that allows for interoperability, customizing, testing, and optimizing of your Zigbee solution.
 
 For detailed documentation of the ZBOSS API and instructions on how to use it, see `ZBOSS API documentation`_.
+
```

Comment:
As per Teams conversation with @wbober , please add a `ZBOSS configuration` section with a mention of the header file:
`For an overview of the default ZBOSS configuration, see :file:`<name of the file>`.
You can also provide information about what this configuration is about and link to configuring Zigbee in NCS...